### PR TITLE
Release notes for Desktop Stable patch

### PR DESCRIPTION
**Pull request by usha-mandya**
### Proposed changes

Release notes for Desktop Stable patch release

### Unreleased project version (optional)

2.2.0.3

### Related issues (optional)

https://docker.atlassian.net/browse/DESKTOP-2380

### CODE REVIEW

File: docker-for-windows/release-notes.md [hunk 1]
```diff
@@ -13,12 +13,47 @@ This page contains information about the new features, improvements, known issue
...
-2020-01-21
+## Docker Desktop Community 2.2.0.3
 
 > [Download](https://hub.docker.com/?overlay=onboarding)
->
-> You must sign in to Docker Hub to download Docker Desktop.
+
+### Upgrades
+
+- [Docker Compose 1.25.3](https://github.com/docker/compose/releases/tag/1.25.3)
```
Comment (mat007) [inline docker-for-windows/release-notes.md]: It’s actually 1.25.4

Comment (usha-mandya) [reply docker-for-windows/release-notes.md]: Fixed.

**Comment by usha-mandya** [comment]
Updated the release date.

File: docker-for-windows/release-notes.md [hunk 2]
```diff
@@ -13,12 +13,49 @@ This page contains information about the new features, improvements, known issue
...
+
+- [Docker Compose 1.25.4](https://github.com/docker/compose/releases/tag/1.25.4)
+- [Go 1.12.16](https://golang.org/doc/devel/release.html#go1.12)
+
+### Bug fixes and minor changes
+
+- Fixed an issue that prevented users from creating files with special characters in the filenames within a shared volume. Fixes [docker/for-win#5520](https://github.com/docker/for-win/issues/5520).
+- Fixed handling of shared volumes with relative paths in `docker-compose.yml`. Fixes [docker/for-win#5516](https://github.com/docker/for-win/issues/5516).
+- Fixed handling of shared volumes where the path case (uppercase / lowercase) did not exactly match the host. Fixes [docker/for-win#5516](https://github.com/docker/for-win/issues/5516).
+- Fixed an issue where changing a file in the Windows file system did not result in the files being updated within the container. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530).
```
Comment (stephen-turner) [inline docker-for-windows/release-notes.md]: add "and 5550".

Comment (usha-mandya) [reply docker-for-windows/release-notes.md]: Fixed.

**Pull request merged**